settings.py

```diff
@@ -179,7 +180,7 @@ def path(*args):
 EXACTTARGET_PASS = config('EXACTTARGET_PASS', None)
 EXACTTARGET_DATA = config('EXACTTARGET_DATA', None)
 EXACTTARGET_OPTIN_STAGE = config('EXACTTARGET_OPTIN_STAGE', None)
-SUPERTOKEN = config('SUPERTOKEN')
+SUPERTOKEN = config('SUPERTOKEN', str(uuid4()))
```

Comment:
Did this so that I didn't have to define any one value in the demo env file that would make them vulnerable on creation.